### PR TITLE
feat(Label): penta updates

### LINKED ISSUE #9981: (Label): Consume core updates for Penta
follow up to core issue https://github.com/patternfly/patternfly/issues/6181

### PR DESCRIPTION
**Pull request by kmcfaul**
**What**: Closes #9981

Added 'orangered' color option & added examples.
Added `isAddLabel` flag, intending for label group's add button.
Added `pf-m-clickable` to non-overflow and non-add label buttons or links.
Added `pf-m-no-padding` to close buttons.
Updated filled, outline, and compact examples to use css for layout instead of manual whitespace.

### CODE REVIEW

**Review by mcoker — APPROVED** [review]
LGTM!

Just a heads up, I have another [PR up in core](https://github.com/patternfly/patternfly/pull/6227) to add status variants, and adds a "filled" modifier to the default/filled labels. It also replaces the info-circle icons we're using as the example icon in labels with a cube icon since the info-ircle icon is used with info labels.

**Review by thatblindgeye — APPROVED** [review]
Looks good. Had a couple comments below, but they aren't blocking to this PR.

- @mcoker the editable label cursor style remains a pointer when the label is currently editable. I'd expect the cursor to be `text` when editing a label (this would be a core followup, unless the intent was to have the pointer cursor)
- Can we open a followup issue to add status support/update non-status labels in example to use the icon Michael mentioned above

**Comment by kmcfaul** [comment]
Opened #10047 to follow up for status labels, in case it doesn't get in on this PR.

**Comment by kmcfaul** [comment]
- Removed `isOverflowLabel` and `isAddLabel`
- Added `overflow` and `add` to `variants`

Since these label types seem to be mutually exclusive with filled and outline. LMK if either should be a flag still instead.

**Review by wise-king-sullyman — APPROVED** [review]

**Comment by lboehling** [comment]
lgtm!

File: packages/react-core/src/components/Label/Label.tsx [hunk 1]
```diff
@@ -58,8 +58,6 @@ export interface LabelProps extends React.HTMLProps<HTMLSpanElement> {
   closeBtnProps?: any;
   /** Href for a label that is a link. If present, the label will change to an anchor element. This should not be passed in if the onClick prop is also passed in. */
   href?: string;
-  /** Flag indicating if the label is an overflow label. */
-  isOverflowLabel?: boolean;
```
Comment (tlabaj) [inline packages/react-core/src/components/Label/Label.tsx:62]: Can you please opne code mode issue for this removed prop.

**Review by tlabaj — APPROVED** [review]
LGTM

**Pull request merged**